### PR TITLE
Changed the package and link where necessary

### PR DESCRIPTION
**Pull request by teryhill**
Changed the package and link where necessary #332 Doing it in smaller chunks to make review easier and prevent stomping

### CODE REVIEW

File: interface/billing/sl_eob_invoice.php [hunk 1]
```diff
@@ -18,11 +18,11 @@
  * You should have received a copy of the GNU General Public License
```
Comment (aethelwulffe) [inline interface/billing/sl_eob_invoice.php]: sl_....argh!  One day we will be rid of it!

File: interface/code_systems/dsmiv_howto.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
  * You should have received a copy of the GNU General Public License
```
Comment (aethelwulffe) [inline interface/code_systems/dsmiv_howto.php]: Delete this file.  It is based on the bad assumption that DSM4 codes actually exist.  It is DSM5 now anyway, and it only refers to ICD10...DSM is a guide, not a code set.

File: interface/code_systems/icd9_howto.php [hunk 3]
```diff
@@ -15,7 +15,7 @@
  * You should have received a copy of the GNU General Public License
```
Comment (aethelwulffe) [inline interface/code_systems/icd9_howto.php]: Snuff this file.

File: interface/eRxGlobals.php [hunk 4]
```diff
@@ -15,193 +15,193 @@
  * General Public License along with this program.
```
Comment (aethelwulffe) [inline interface/eRxGlobals.php]: Would live to see eRX go into the modules dir...some day.

File: interface/main/onotes/office_comments.php [hunk 5]
```diff
@@ -13,7 +13,7 @@
  * You should have received a copy of the GNU General Public License
```
Comment (aethelwulffe) [inline interface/main/onotes/office_comments.php]: Do onotes work?

**Comment by teryhill** [comment]
No code change just changes to link and package and reference to libreehr

**Comment by teryhill** [comment]
@tmccormi  Can you review this?

**Pull request merged**